### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by alice-hannah** [ScotGovAnalysis/rap-self-assessment#8]
- [ ] Focus still on form when disconnected from server. 
   - Have to flick through all other interactive elements before focus returns to 'Reload'. 
   - Could be confusing as any form input will be lost when reloaded.

- [x] Focus skips callout blocks ('what does this mean?')
   - Try using `<details>` component instead of Quarto callout?

- [x] Focus not in logical order. Next logical step after completing form would be to download report.
   - Tried sidebar as don't think all users will want to complete full form. Wanted users to be able to download at any point without having to scroll through whole form. 
   - An alternative could be to make each RAP level a collapsible section?

- [ ] Hidden labels for dropdown form not visible by screen reader
   - Something to do with ARIA property
   - Think this can be resolved by setting `selectize = FALSE` in `selectInput` (however this then means arrow disappears from RHS)
